torchvision/io/__init__.py

```diff
@@ -6,6 +6,7 @@
 from ._video_opt import (
     Timebase,
     VideoMetaData,
+    _HAS_VIDEO_DECODER,
```

Comment:
nit: this is not a great name for this attribute.

I would maybe name it as `_HAS_GPU_VIDEO_DECODER` instead.

But this can be changed in a follow-up PR.

Reply:
https://github.com/pytorch/vision/issues/5148